test/e2e/flowaggregator_test.go

```diff
@@ -456,26 +460,62 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// Creating test client and server Pods for ToExternal tests
+	serverPodPort := int32(80)
+	_, serverIPs, cleanupFunc := createAndWaitForPod(t, data, func(name string, ns string, nodeName string, hostNetwork bool) error {
+		return data.createServerPod(name, data.testNamespace, "", serverPodPort, false, true)
+	}, "test-server-", "", data.testNamespace, false)
+	defer cleanupFunc()
+
+	// Deploy the client Pod on the control-plane node
+	clientName, clientIPs, cleanupFunc := createAndWaitForPod(t, data, data.createBusyboxPodOnNode, "test-client-", nodeName(0), data.testNamespace, false)
+	defer cleanupFunc()
+
+	// ToExternalEgressPolicy tests the export of IPFIX flow records when a source Pod
+	// sends traffic to an external IP and an Egress Policy applied on the source Pod
+	t.Run("ToExternalEgressPolicy", func(t *testing.T) {
```

Comment:
do you think we can check both cases: Egress Node == Source Node and Egress Node != Source Node?